packages/flutter/lib/src/material/text_field.dart

```diff
@@ -648,21 +654,71 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
     _cancelCurrentSplash();
   }
 
-  void _handleSingleLongTapDown() {
+  void _handleSingleLongTapStart(GestureLongPressDragStartDetails details) {
     if (widget.selectionEnabled) {
+      // We can't break the API for SelectionChangedCause now.
```

Comment:
This comment is a little confusing, maybe it doesn't belong here? Someone reading the code will probably not understand "we can't break the API ..." I'm still not sure exactly what this comment is trying to clear up.